def raise_exception_msg(ms):
    raise NameError(ms)
